app/src/main/java/org/p2p/wallet/receive/tokenselect/ReceiveTokensPresenter.kt

```diff
@@ -57,6 +62,24 @@ class ReceiveTokensPresenter(
         }
     }
 
+    private fun preparePinedWormholeTokens(): List<TokenData> {
+        return ERC20Tokens.values().mapNotNull { erc20Token ->
+            interactor.findTokenDataByAddress(erc20Token.mintAddress)?.let { token ->
+                TokenData(
+                    mintAddress = token.mintAddress,
+                    name = erc20Token.replaceTokenName ?: token.tokenName,
+                    symbol = erc20Token.replaceTokenSymbol ?: token.tokenSymbol,
+                    iconUrl = erc20Token.tokenIconUrl ?: token.iconUrl,
+                    decimals = token.decimals,
+                    coingeckoId = erc20Token.coingeckoId,
+                    isWrapped = false,
+                    serumV3Usdc = null,
+                    serumV3Usdt = null
+                )
```

Comment:
can we do this mapping inside the interactor?
Do we have `EthereumInteractor`?

Reply:
it is in Ethereum package and here we need TokenData that we can not access from EthereumInteractor

Reply:
looks like we need to move `TokenData` to core as well. 
But it's optional